docs/taker.md

```diff
@@ -116,45 +118,175 @@ SUBCOMMANDS:
+- **regular**: All single signature regular wallet coins (seed balance)
+- **spendable**: Total spendable amount in wallet (regular + swap balance)
+- **swap**: All 2of2 multisig coins received in swaps
+
+### List All UTXOs
+
+To view all UTXOs in your wallet, use this command:
+
+```bash
+$ taker -r 127.0.0.1:38332 -a user:password list-utxo
+```
+
+**Output:**
+
+```bash
+```
+
+This lists all UTXOs we know about along with their spend info. Since the wallet is empty, no UTXOs are displayed.
+
+### List Regular UTXOs
+
+To view only single signature wallet UTXOs, run:
+
+```bash
+$ taker -r 127.0.0.1:38332 -a user:password list-utxo-regular
+```
 
-Now we are ready to initiate a coinswap. We are first going to sync the offer book to get a list of available makers.
+**Output:**
 
-```sh
-$ taker -r 127.0.0.1:38332 -a user:pass fetch-offers
+```bash
+```
+
+This lists all single signature wallet UTXOs. These are all non-swap regular wallet UTXOs.
+
+### List Swap UTXOs
+
+To view UTXOs received from incoming swaps, run:
+
+```bash
+$ taker -r 127.0.0.1:38332 -a user:password list-utxo-swap
+```
+
+**Output:**
+
+```bash
+```
+
+This lists all UTXOs received in incoming swaps. Since we haven't performed any coinswaps yet, this list is empty.
+
+### List Contract UTXOs
+
+To check for any locked funds from failed swaps, run:
+
+```bash
+$ taker -r 127.0.0.1:38332 -a user:password list-utxo-contract
+```
+
+**Output:**
+
+```bash
+```
+
+This lists all UTXOs that we need to claim via timelock. If you see entries in this list, you should run the `recover` command to claim them.
+
+### Fetch Available Offers
+
+Now we are ready to initiate a coinswap. We are first going to sync the offer book to get a list of available makers:
+
+```bash
+$ taker -r 127.0.0.1:38332 -a user:password fetch-offers
+```
+
+**Output:**
+
+```json
+{
+  "address": "m3qn53qt3wukwqvbx4yhq7v2qrsvf2oiddtjsqigwktzuuot33hmmuad.onion:8302",
+  "amount_relative_fee_pct": 0.1,
+  "base_fee": 100,
+  "fidelity_bond": {
+    "amount": 50000,
+    "cert_expiry": 11,
+    "conf_height": 12859,
+    "lock_time": 25962,
+    "outpoint": {
+      "txid": "241a007ae754ac41b1910bead088b6f83a3bab2a4dc77118568d55e3a84dbe05",
+      "vout": 0
+    }
+  },
+  "max_size": 1949696,
+  "min_size": 10000,
+  "minimum_locktime": 20,
+  "required_confirms": 1,
+  "time_relative_fee_pct": 0.005,
+  "timestamp": 1746899012,
+  "tweakable_point": "0295dc108b6fe6036072f7db591736f69296a9312b1b95d986a8f88fa0672cd5d4"
+}
 ```
 
-This will fetch the list of available makers from the directory server. Now we can initiate a coinswap with the makers.
+This will fetch the list of available makers and display their offers.
 
-```sh
-$ taker -r 127.0.0.1:38332 -a user:pass coinswap
+### Initiate a Coinswap
+
+Now we can initiate a coinswap with the makers:
+
+```bash
+$ taker -r 127.0.0.1:38332 -a user:password coinswap
```

Comment:
Can we show some coinswap logs here. Just show the starting few lines, and ending few lines, with dots in between.